### ISSUE TITLE
Configure leapwork plugin on a https-based jenkins

### ISSUE THREAD

**Issue by srindal** [Customatics/leaptest-plugin#28]
Hello

I'm in the middle of configuring the leapwork-plugin on a new Jenkins. However when I set up a job and click "select schedules" I receive a general browser error, saying the either my url, accesskey or something else is wrong. I've checked the developer-extensions and find that i'm running into a "Same-origin" policy issue. I have tried to install the CORS plugin and configure it, but since leapwork is running http and jenkins is on https it does not work. 

Does anybody else have this issue?

**Comment by romanchetto** [comment]
Hello @srindal 
Maybe, disabling CSRF protection at Jenkins Global Security configuration may help. Unfortunately, I'm not able to check it on HTTPS Jenkins.
![CSRF](https://user-images.githubusercontent.com/22563434/69610919-d4f38500-103d-11ea-8657-2b25d91c4e1f.PNG)
Also, if you have get such error as on the image below, check "Leapwork Integration" controller settings values: hostname, port (by default is 9001) and Access key
![Jenkins error](https://user-images.githubusercontent.com/22563434/69612034-01100580-1040-11ea-8851-7c091962182d.PNG)

**Comment by srindal** [comment]
Thanks for the answer, but I'm stilling getting "same origin policy" errors. I'm considering setting up an nginx proxy that maps the http://leapworkcontroller:9001 to an address in my jenkins-url.
![image](https://user-images.githubusercontent.com/9130818/69615394-12074980-1035-11ea-8ae9-9200f23928ec.png)

**Comment by romanchetto** [comment]
@srindal 
It looks like "same-origin" policy blocks the request. Considering info about it:
https://en.wikipedia.org/wiki/Same-origin_policy
Maybe, this may help you to handle this policy and sent HTTP requests
https://wiki.jenkins.io/display/JENKINS/Cors+Filter+Plugin

**Comment by srindal** [comment]
Hi. I've have tried the CORS plugin. The problem is with bridging from Https - which my jenkins is running and to http which is the leapwork controller protocol.

**Comment by romanchetto** [comment]
@srindal Have you tried to install CORS extension for browser?

**Comment by srindal** [comment]
No, but I don't like the idea. The system should work without any client tricks

**Comment by romanchetto** [comment]
@srindal I'm not sure that it is possible without any tricks. At current moment I don't have any more ideas. Leaving this issue opened if somebody knows how to make it work.

**Comment by bicschneider** [comment]
As I recall.. The problem is not on the Jenkins side but on LeapWork server. You cannot from a secure site(Jenkins) retrieve data from a insecure source(LeapWork server).
